kits/agentic/code-review/README.md

```
@@ -0,0 +1,185 @@
+
+## Environment Variables
+
+Create a `.env` file in the kit root:
+
+---
+
+## Lamatic Flow Setup
+
+This kit requires a Lamatic flow with the following node structure:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)